docs/HashHistoryCaveats.md

```diff
@@ -16,6 +16,18 @@ let history = createHistory({
 let history = createHistory({
   queryKey: false
 })
+
+// Use custom URL transformations, for example for hashbang support
+let history = createHistory({
+  transformPath: (path, encode) => {
```

Comment:
Separate encode/decode (or transform/restore/whatever) callbacks would be cleaner than a single callback with a boolean flag IMO.